pkg/validator/schema.go

```diff
@@ -77,7 +80,7 @@ func parseCheck(rawBytes []byte) (config.SchemaCheck, error) {
 	}
 }
 
-func resolveCheck(conf *config.Configuration, checkID string, controller kube.GenericWorkload, container *corev1.Container, target config.TargetKind, isInitContainer bool) (*config.SchemaCheck, error) {
+func resolveCheck(conf *config.Configuration, checkID string, controller *kube.GenericWorkload, container *corev1.Container, ingress *v1beta1.Ingress, target config.TargetKind, isInitContainer bool) (*config.SchemaCheck, error) {
```

Comment:
I think we can modify the function signature to make this a little more future proof:
```
resolveCheck(
  conf *config.Configuration,
  checkID string,
  target config.TargetKind,
  metav1.ObjectMeta,
  containerName string,
  isInitContainer bool)
```

so we'll just be passing in names/metadata rather than the underlying objects.

No worries if that's a pain though